Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -6,6 +6,7 @@
 void SFilledGraphWidget::Construct(const FArguments& InArgs)
 {
 	InData = InArgs._InDataArray;
```

Comment:
I feel it's probably the right time to delete the InData arg and the general '1D' data handling as your new code allows the end user to make the same plot if they supply the appropriate X values. Or is there a reason for keeping it around still?